2022q4/FreshPorts.adoc

```diff
@@ -1,7 +1,8 @@
 ## FreshPorts - help wanted ##
 
-Link:	 https://freshports.org/[FreshPorts]
-Link:	 https://news.freshports.org/[FreshPorts blog]
+Links: +
+https://freshports.org/[FreshPorts] URL: https://freshports.org/[https://freshports.org/] +
+https://news.freshports.org/[FreshPorts blog] URL: https://news.freshports.org/[https://news.freshports.org/]
 
 Contact: Dan Langille <dan@FreeBSD.org>  
```

Comment:
I sent an e-mail to Dan about this. I'm more familiar with him as `dvl@` (in the FreshPorts page for acme.sh, and so on).

Reply:
> I sent an e-mail to Dan about this. I'm more familiar with him as `dvl@` (in the FreshPorts page for acme.sh, and so on).

I'll defer to him on the rest of this.